<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>384</width>
                <height>435</height>
            </rect>
        </property>
        <property name="font">
            <font>
                <family>SolaimanLipi</family>
                <pointsize>12</pointsize>
            </font>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLineEdit" name="lddate">
                    <property name="placeholderText">
                        <string>তারিখ</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QComboBox" name="comboBox">
                    <property name="font">
                        <font>
                            <family>SolaimanLipi</family>
                            <pointsize>14</pointsize>
                        </font>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="ldamount">
                    <property name="font">
                        <font>
                            <pointsize>14</pointsize>
                        </font>
                    </property>
                    <property name="placeholderText">
                        <string>টাকার পরিমান (৳)</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QTextEdit" name="tddesc">
                    <property name="placeholderText">
                        <string>মন্তব্য</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="label_msg">
                    <property name="text">
                        <string>শেষ সংযোজনঃ নেই</string>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QRadioButton" name="rb_add_more">
                            <property name="text">
                                <string>আরো যোগ করুন</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pb_ok">
                            <property name="text">
                                <string>OK</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pb_cancel">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
